nebula-components/objects/_ui-list.scss

```diff
@@ -0,0 +1,15 @@
+@if $nb-use-ui-list {
+  .#{$nb-namespace}o-ui-list {
+    @include nb-ui-list;
+  }
+
+  .#{$nb-namespace}o-ui-list__item--divider {
```

Comment:
Just like the `bare-list` I'd go with `o-ui-list__item` as the className as this isn't just a divider, it contains content.

Reply:
done